backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -4,12 +4,15 @@
 import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
 import javax.persistence.OneToMany;
-import javax.persistence.Transient;
+import lombok.Getter;
 import org.springframework.boot.context.properties.ConstructorBinding;
 
 /** The durable (and non-deletable) representation of a POC test device model. */
 @Entity
+@Getter
```

Comment:
Do we use Lombok annotations on other database models? I'm not sure if it will interact with Hibernate in any unexpected ways.